sharding-core/sharding-core-route/src/main/java/org/apache/shardingsphere/core/route/router/sharding/condition/generator/impl/ConditionValueBetweenOperatorGenerator.java

```diff
@@ -39,7 +42,15 @@
     public Optional<RouteValue> generate(final PredicateBetweenRightValue predicateRightValue, final Column column, final List<Object> parameters) {
         Optional<Comparable> betweenRouteValue = new ConditionValue(predicateRightValue.getBetweenExpression(), parameters).getValue();
         Optional<Comparable> andRouteValue = new ConditionValue(predicateRightValue.getAndExpression(), parameters).getValue();
-        return betweenRouteValue.isPresent() && andRouteValue.isPresent()
-                ? Optional.<RouteValue>of(new RangeRouteValue<>(column.getName(), column.getTableName(), Range.closed(betweenRouteValue.get(), andRouteValue.get()))) : Optional.<RouteValue>absent();
+        if (betweenRouteValue.isPresent() && andRouteValue.isPresent()) {
+            return Optional.<RouteValue>of(new RangeRouteValue<>(column.getName(), column.getTableName(), Range.closed(betweenRouteValue.get(), andRouteValue.get())));
+        }
+        DefaultTimeService defaultTimeService = new DefaultTimeService();
```

Comment:
Do we need SPI to load `TimeService`?

Reply:
```
sharding clomn is create_time
select * from ts_order where create_time between ? and now()
```
I think if use this to query , we still need SPI to load `TimeService`

Reply:
What I mean is we need SPI to load `TimeService`, but we should load SPI obvious from the caller.
Right now, we put loading SPI logic into `DefaultTimeService`, that make `DefaultTimeService` to complicated. 
`DefaultTimeService` may just as a regular SPI implementation.

Reply:
I get what you meant now